person.rb

```diff
@@ -0,0 +1,29 @@
+# class for person
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
[Optional] It would be better to generate the `id` randomly rather than make it `unknown`  since it will be useful if you want to automate the process of generating unique id values and provide sensible default values for the other attributes. You can visit this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more.
So you just need to remove the `id` from the constructor and change line 6 to `@id = Random.rand(1..1000)` to generate the id randomly.